classifier/__init__.py

```diff
@@ -92,7 +92,7 @@ class Classifier(object):
     AUDIENCES = set([AUDIENCE_ADULT, AUDIENCE_ADULTS_ONLY, AUDIENCE_YOUNG_ADULT,
                      AUDIENCE_CHILDREN, AUDIENCE_ALL_AGES, AUDIENCE_RESEARCH])
     AUDIENCES_NO_RESEARCH = [
-        x for x in AUDIENCES if x != AUDIENCE_RESEARCH
+        x for x in AUDIENCES if x != "Research"
```

Comment:
Can we avoid spelling out the constant twice?

Reply:
I forgot to point a few lines out I was having trouble updating. Here, `AUDIENCE_RESEARCH` is not defined in the scope of the list comprehension. It's fine if it's used without the `if` statement but I couldn't figure out how to use the variable otherwise.